seatunnel-connectors-v2/connector-doris/src/main/java/org/apache/seatunnel/connectors/doris/source/serialization/RowBatch.java

```diff
@@ -71,21 +75,22 @@
 
 @Slf4j
 public class RowBatch {
```

Comment:
why change `RowBatch`? Any test case?

Reply:
> why change `RowBatch`? Any test case?

I adapted the latest version of doris, and I found that many types were incompatible, so I added some type conversions.It is backwards compatible